map/views.py

```diff
@@ -0,0 +1,28 @@
+from django.http import JsonResponse
+from events.models import Event
+from location.models import Location
+from django.shortcuts import render
+from django.core.serializers import serialize
+import json
+
+
+def show_map(request):
+    # return render(request, 'show_map.html')
+    return render(request, "show_map.html")
```

Comment:
The UI looks great!! I wonder if it's possible that the map can be integrated into the event page so that we don't need to go to a separate "map" page to see the map. Or is this intended for next PR? (A possible reference is like this: 
![image](https://github.com/gcivil-nyu-org/Wednesday-Fall2023-Team-3/assets/130470692/4fef79af-6cb8-4455-8020-94d352a94892)

Reply:
Yes, that'll be done in the next one